Lab2.ipynb

```diff
@@ -0,0 +1,608 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "xlow = 0\n",
+    "xhigh = 10\n",
+    "xspace = 15\n",
+    "\n",
+    "x = np.linspace(xlow,xhigh,xspace)\n",
+    "\n",
+    "ray_dist = stats.rayleigh.pdf(x,loc=0)\n",
+    "ray_dist = ray_dist/sum(ray_dist)\n",
+    "comb = ray_dist\n",
+    "k=2000\n",
+    "for i in range(k):\n",
+    "    comb = np.convolve(comb,expo_dist)\n",
+    "    \n",
+    "cx = np.linspace(xlow,xhigh,len(comb))\n",
+    "plt.step(cx,comb)\n",
+    "plt.yscale('log')\n",
+    "plt.xlabel('Value')\n",
+    "plt.ylabel('Probability')\n",
+    "plt.title('Probability Density Function for Rayleigh Distribution, 2000 Intervals, Averaged Distribution')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The shape of the exponential distribution does indeed change after averaging over many trials. When comparing the averaged distributions to that of the original, it is clear that the distribution eventually bends toward a Gaussian distribution. Took around 2000 iterations to get close to a Gaussian for this Rayleigh distribution."
```

Comment:
I think you mean to say the shape of the Rayleigh changes because that is the distribution you are looking at, not the exponential. Also I think you could explain a bit more about what specific features of the distributions change as you average over more intervals.